### DISCUSSION TITLE
Is there a way to get junit reports.xml or xctestresult file

### DISCUSSION THREAD

**Discussion by badrinrs** [danielpaulus/go-ios#288] [Q&A]
As the question above or any kind of reporting file which details which tests passed or failed and provides more insights into what failed.

**Comment by nanoscopic** [comment]
I don't believe go-ios or pmd3 currently implement collecting the results that are streamed out by the phone.

You may be able to construct such a thing yourself by looking at the CLI output from go-ios when a test is run.

Actual handling of the results and/or construction of the sort of result file given by xcode is not a thing yet. Good idea though. You should submit this as a feature request issue.
